shared/schemas/workflow-option-schema.json

```diff
@@ -0,0 +1,28 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "WorkflowOption",
```

Comment:
I think the name needs to be changed. what does workflow option mean?

Reply:
done. We use `outputs` now